backend/jobs/metric-alerts/metric-alerts.go

```diff
@@ -188,45 +195,78 @@ func processMetricAlert(ctx context.Context, DB *gorm.DB, MailClient *sendgrid.C
 		groupByKey = groupBy[0]
 	}
 
+	bucketsInner := buckets.Buckets
```

Comment:
Not sure if this is a local issue, but getting a null pointer reference trying to start the backend here
```
github.com/highlight-run/highlight/backend/jobs/metric-alerts.processMetricAlert({0x1060466d0, 0x10a817160}, 0x140030923f0, 0x1400700e050, 0x14005f4db00, 0x14006dbaa20, 0x0)
	/Users/spenceramarantides/repos/highlight/backend/jobs/metric-alerts/metric-alerts.go:198 +0x14cc
github.com/highlight-run/highlight/backend/jobs/metric-alerts.WatchMetricAlerts.func1.1()
	/Users/spenceramarantides/repos/highlight/backend/jobs/metric-alerts/metric-alerts.go:50 +0x68
github.com/highlight-run/workerpool.(*WorkerPool).SubmitRecover.func1()
```

Reply:
I might just have an env variable set differently for myself, but I think this is coming from the `lambdaClient` usage (which is probably nil locally). will remove the lambda client here anyway since I refactored that when I switched over to a flask app